addons/l10n_id_efaktur_coretax/views/account_move.xml

```diff
@@ -26,6 +26,9 @@
                 <field name="l10n_id_coretax_document" readonly="True"/>
             </xpath>
             <xpath expr="//field[@name='l10n_id_csv_created']" position="replace"/>
+            <xpath expr="//field[@name='tax_totals']" position="attributes">
```

Comment:
You should make the original field invisible for ID, and add a new <field> with the widget only for ID